recipe/build.sh

```diff
@@ -1,27 +1,33 @@
 #!/bin/bash
 
+set -eo pipefail
+
 declare -a CMAKE_PLATFORM_FLAGS
 if [[ ${HOST} =~ .*linux.* ]]; then
     CMAKE_PLATFORM_FLAGS+=(-DCMAKE_TOOLCHAIN_FILE="${RECIPE_DIR}/cross-linux.cmake")
```

Comment:
I would remove this part. It's not a pattern we need to use as far as I can tell.

Reply:
Removed